src/lib/i18n/setup.ts

```diff
@@ -77,6 +68,13 @@ export const i18nSetup = async () => {
             nonExplicitSupportedLngs: true,
             cleanCode: true,
             lowerCaseLng: true,
+          })
+          .then(() => {
```

Comment:
why 'then'?

the init is awaited, just add a statement after it.
I don't like mixing async with promises patterns.

have you checked that the init() return the 't' object?

I see that in respect to the old 'make', the new one has not the keyPrefix parameters. is it correct or an oversight?